README.md

```diff
@@ -25,11 +25,26 @@ To use any of the Babel client functions, you must have a Persona token. Read th
 out the [Persona node client](https://github.com/talis/persona-node-client). You might also want to look at the [Babel docs](http://docs.talisbabel.apiary.io/) too.
 
 ### Target Feeds
-Get a feed based on a target
+Make a HEAD request to see if any annotations have been added to a feed
 ```javascript
 var target = 'stay-on-target';
 var token = req.personaClient.getToken(req);
-babelClient.getTargetFeed(target, token, {}, function(error, results){
+babelClient.headTargetFeed(target, token, {delta_token:1}, function(error, response){
+    // do stuff
+    var newItemsFound = response.headers['x-feed-new-items'];
+    // newItemsFound will now show you how many annotations have been added since the annotation with a delta_token equal to 1.
+});
+```
+
+Get a feed based on a target. Now supports passing through of `offset`, `limit` and `delta_token` in params.
+```javascript
+var target = 'stay-on-target';
+var token = req.personaClient.getToken(req);
+var hydrate = true;
+var params = {
+    delta_token:'123'
+};
+babelClient.getTargetFeed(target, token, hydrate, params, function(error, results){
     // do stuff
```

Comment:
It might be nice to have a simple example of what to do with the results e.g. what format they come back in?